cluster/gce/gci/configure-helper.sh

```diff
@@ -341,6 +355,15 @@ function start-kubelet {
       flags+=" --hairpin-mode=${HAIRPIN_MODE}"
     fi
   fi
+  # Network plugin
+  if [[ -n "${NETWORK_PROVIDER:-}" ]]; then
+    flags+=" --network-plugin-dir=/home/kubernetes/bin"
```

Comment:
True. Will fix it along with: https://github.com/kubernetes/kubernetes/issues/28563